Contribution

SidWorks opened the pull request:
The Contribution.md had legacy links Instruction for using GitBook was also not wrong!
Updated the links to the working ones and added new instruction to use GitBook  along with gif for better clarity.

KINGSABRI commented:
Hi @SidWorks 
The changes are huge and may need some time. Would you please brief me about the changes?

SidWorks commented:
> Hi @SidWorks
> The changes are huge and may need some time. Would you please brief me about the changes?
> What I see is
> 
> * Deleting node_modules
> * Deleting styles
> 
> I understand these were for the old version of Gitbook but I didn't delete it because I don't know if any of it still in use
> 
> Thanks

The links that were mention gor GitBooks were not working! I changed to the new one.
Using GitBook instruction was old, updated that to a newer one with Gif.

Node modules and styles are untouched.

KINGSABRI commented:
Thank you for your contribution
